src/main/java/com/global/api/builders/ManagementBuilder.java

```diff
@@ -330,6 +339,16 @@ public ManagementBuilder withTransactionMatchingData(TransactionMatchingData val
         transactionMatchingData = value;
         return this;
     }
+    
+    public ManagementBuilder withEstimatedTransactions(Integer estNumTxn) {
+        this.estimatedNumTxn = estNumTxn;
+        return this;
+    }
+    
+    public ManagementBuilder isFinal(boolean isFinal) {
+        this.isFinal = isFinal;
+        return this;
+    }
```

Comment:
These might be better as a single method with a data object to pass the options, e.g.:

```java
public ManagementBuilder withMultiCapture() {
    return this.withMultiCapture(null);
}
public ManagementBuilder withMultiCapture(MultiCaptureOptions options) {
   // implementation
   return this;
}
```

@russelleverett thoughs?

Reply:
I noticed some kind of pattern where mandatory request parameters were required when calling the method, and optional request parameters being added using withXXX methods.
What is your actual way of doing it?

Reply:
@Shane I agree we should have a single method. Would fit more with the style of the SDK. Though depending on the number of options these may just be direct parameters rather then an option object.